services/help-service.ts

```diff
@@ -153,7 +202,20 @@ export class HelpService implements IHelpService {
 		const mdFileName = await this.convertCommandNameToFileName(commandName) + HelpService.MARKDOWN_FILE_EXTENSION;
 		this.$logger.trace("Reading help for command '%s'. FileName is '%s'.", commandName, mdFileName);
 
-		const markdownFile = _.find(this.$fs.enumerateFilesInDirectorySync(this.pathToManPages), file => path.basename(file) === mdFileName);
+		let markdownFile = _.find(this.$fs.enumerateFilesInDirectorySync(this.pathToManPages), file => path.basename(file) === mdFileName);
+		if (!markdownFile) {
+			// Try getting help from extensions
+			const installedExtensionsData = this.$extensibilityService.getInstalledExtensionsData();
+			for (const extensionData of installedExtensionsData) {
+				if (extensionData.docs) {
+					markdownFile = _.find(this.$fs.enumerateFilesInDirectorySync(extensionData.docs), file => path.basename(file) === mdFileName);
```

Comment:
You can extract `file => path.basename(file) === mdFileName` lambda to a separate `const` variable and reuse it here and up above on line 205